tests/dev-pty-yy.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Check printing of the counter endpoint of /dev/ptmx in -yy mode.
+ *
+ * Copyright (c) 2024 The strace developers.
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#include "tests.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <sys/ioctl.h>
+
+int
+main(void)
+{
+	int ptmx_fd = posix_openpt(O_RDWR);
+	if (ptmx_fd < 0)
+		perror_msg_and_fail("posix_openpt");
+
+	if (unlockpt(ptmx_fd) < 0)
+		perror_msg_and_fail("unlockpt");
+
+	char *pts = ptsname(ptmx_fd);
+	if (pts == NULL)
+		error_msg_and_fail("failed in ptsname()");
```

Comment:
Since all these errors are framework issues,  I suggest skipping the test instead of failing it.